tls/s2n_client_hello.c

```diff
@@ -299,11 +330,35 @@ int s2n_process_client_hello(struct s2n_connection *conn)
     return S2N_SUCCESS;
 }
 
+static S2N_RESULT s2n_client_hello_process_cb_response(struct s2n_connection *conn, int rc)
+{
+    if (rc >= 0) {
```

Comment:
this rc is response from the callback which is returned by clients, and that API doesn't mention/enforce using S2N_SUCCESS and instead mentions returning 0 for success. Therefore thought it would be better to use `0` instead of `S2N_SUCCESS`